core/game/game.go

```diff
@@ -13,6 +22,43 @@ func New() *Game {
 	return g
 }
 
+// TreePath returns the node found at a given path
+func (g *Game) TreePath(p Path) (*Node, error) {
```

Comment:
I think treepath logic should largely live in core/treepath/treepath.go rather than directly in game. It has functionality that is overlapping with but not entirely concurrent with games.